### DISCUSSION TITLE
the "basic" example does not have a "setter"

### DISCUSSION THREAD

**Discussion by granadacoder** [redis/lettuce#1614] [Q&A]
Is there a reason why the "setter" is missing?  Or is it just a small oversight?

https://github.com/lettuce-io/lettuce-core/wiki/Basic-usage

RedisCommands<String, String> commands = connection.sync();            (3)

commands.set("foo", "bar");  << Is there a reason why this is missing?

String value = commands.get("foo");                                    (4)

**Comment by mp911de** [comment]
[`RedisCommands`](https://lettuce.io/core/release/api/io/lettuce/core/api/sync/RedisCommands.html) extends [`RedisStringCommands`](https://lettuce.io/core/release/api/io/lettuce/core/api/sync/RedisStringCommands.html#set-K-V-) which defines the `set(K, V)` method. 

That being said, it's not a setter in the Pojo sense.
